client/index.html

```diff
@@ -241,29 +241,28 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Notifications</h2>
 							</div>
-							<div class="col-sm-12">
+							<div class="col-sm-6">
 								<label class="opt">
 								<input id="badge" type="checkbox" name="badge">
 								Enable badge
 								</label>
-							</div>
-							<div class="col-sm-12">
```

Comment:
I don't have a strong opinion on this. (However I find it ugly whatever the solution but that's not your fault)

I'm fine with that so you can keep your change, but at some point, when we properly discuss configuration (or even before that), a UX/UI pass on the settings page will be much appreciated!